bin/paypal-settlement/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/brave-intl/bat-go/settlement/paypal"
+	"github.com/shopspring/decimal"
+)
+
+var input = flag.String("in", "", "the file or comma delimited list of files that should be utilized")
+var currency = flag.String("currency", "", "a currency must be set")
+var auth = flag.String("auth", "", "an authorization bearer token must be set")
+var date = flag.String("date", "", "the date of the batch")
```

Comment:
why do I need to pass a date?

Reply:
i believe i started to use this to create a execution date, opted for a time.now()